### ISSUE TITLE
telescope selection fails

### ISSUE THREAD

**Issue by matu3ba** [marko-cerovac/material.nvim#129]
Selection of any style in the telescope menu breaks with the following message:
```
E5108: Error executing lua ...aterial.nvim/lua/material/functions/telescope_styles.lua:25: attempt to call
 field 'clear_theme' (a nil value)                                                                        
stack traceback:                                                                                          
        ...aterial.nvim/lua/material/functions/telescope_styles.lua:25: in function 'key_func'            
        ...k/packer/start/telescope.nvim/lua/telescope/mappings.lua:341: in function 'execute_keymap'     
        [string ":lua"]:1: in main chunk 
```

Changing the scheme once does not change the behavior.

**Comment by marko-cerovac** [comment]
Sorry, my bad. I left some dead code around. Should be fixed now.

**Comment by matu3ba** [comment]
Nice, that was quick. Fixed in e4ff40aa2a8ad60742baca6aa852abbd304c5ee5.
